core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/ExpansionGreedyScheduleGenerator.scala

```diff
@@ -202,6 +212,9 @@ class ExpansionGreedyScheduleGenerator(
     // mark links that go to downstream regions
     populateDependeeLinks(regionDAG)
 
+    // add both materialized output ports and view-result output ports to regions
+    updatesRegionsWithOutputPortStorage(outputPortsToMaterialize.toSet, regionDAG)
```

Comment:
use `update` insted of `updates`? don't need the `s` in method name.

Reply:
Fixed.